frontend/src/react/Copy/AnswerExercice/AnswerQuestion/Question.js

```diff
@@ -111,33 +115,30 @@ export default class Question extends Component {
     const {showScale, question, index} = this.props;
     const {err, saved} = this.state;
     return (
-      <>
-        {
-          err && <Error errors={err} status={err.response.status}/>
-        }
-        <QuestionDisplayer
-          question={question}
-          showScale={showScale}
-          index={index}
-        />
-        <div className="box">
-          <AnswerDisplayer
-            editorState={this.state.editorState}
-            myBlockStyleFn={this.myBlockStyleFn}
-            handleEditorClick={this.handleEditorClick}
-            handleChange={this.handleChange}
-            handleKeycommand={this.handleKeyCommand}
-            handleBlur={this.handleControllerClick}
-            onUnderlineClick={this.onUnderlineClick}
-            onBoldClick={this.onBoldClick}
-            onItalicClick={this.onItalicClick}
-          />
-          <ControllerDisplayer
-            handleControllerClick={this.handleControllerClick}
-            saved={saved}
-          />
-        </div>
-      </>
+        <>
+            {
+            !this.state.readOnly && err && <Error errors={err} status={err.response.status}/>
+            }
+            <QuestionDisplayer question={question}
+                               showScale={showScale}
+                               index={index}/>
+            <div className="box">
+                <AnswerDisplayer editorState={this.state.editorState}
+                                 myBlockStyleFn={this.myBlockStyleFn}
+                                 handleEditorClick={this.handleEditorClick}
+                                 handleChange={this.handleChange}
+                                 handleKeycommand={this.handleKeyCommand}
+                                 handleBlur={this.handleBlur}
```

Comment:
Did you test? As far as I know, I removed the handleBlur method and the handleBlur prop points to handleControllerClick @NansD

Reply:
Wups
Actually you can pass props to a component that doesn't use them, it's just useless

Reply:
I know, but I use that props :) Just not the same handler. So now it's broken

Reply:
Alright I'm making a branch